gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/ViewInfo.java

```diff
@@ -144,6 +145,16 @@ ToStringHelper toStringHelper() {
   }
 
   @Override
+  public boolean equals(Object obj) {
+    return obj instanceof ViewInfo && Objects.equals(toPb(), ((ViewInfo) obj).toPb());
```

Comment:
Nit, you may want to consider renaming the base class, BaseTableInfo, equals and hashCode to
`baseEquals` and `baseHashCode`. Also I think `baseEquals` can work for all if you write it this way:
```java
  return obj != null && getClass().isInstance(obj) && Objects.equals(toPb(), ((BaseTableInfo) obj).toPb());
```